### ISSUE TITLE
Training loss is unstable

### ISSUE THREAD

**Issue by Kizna1ver** [baaivision/EVA#125]
Hi @Yuxin-CV,
I am trying to reproduce EVA2's MIM pretraining on my own dataset, but I got this loss peak. Based on several days of empirical results, this often results in poorer performance on downstream tasks. I tried lowering the learning rate, which solved the problem when the learning rate was relatively small, but this made the model converge slower. Are there any other methods that can help solve the issue of loss instability during training?
<img width="1629" alt="image" src="https://github.com/baaivision/EVA/assets/60084536/a7dc7ab4-189a-467f-91aa-84488953b837">

**Comment by leng-yue** [comment]
Is a negative loss usual in eva model?

**Comment by mactavish91** [comment]
Hello, I have encountered a similar problem. How did you solve it？

**Comment by Kizna1ver** [comment]
> Is a negative loss usual in eva model?

Yes, a negative cosine similarity loss is used in the MIM training stage.

**Comment by Kizna1ver** [comment]
> Hello, I have encountered a similar problem. How did you solve it？

I tried to increase the learning rate while reducing the Adam beta2 parameters. Although there are still loss spikes, they last for shorter steps and can change back to the previous value more quickly.

**Comment by leng-yue** [comment]
Thanks a lot.
